src/library_async.js

```diff
@@ -462,6 +462,7 @@ mergeInto(LibraryManager.library, {
     return Asyncify.handleSleep((wakeUp) => safeSetTimeout(wakeUp, ms));
   },
 
+#if !WASMFS
```

Comment:
Can we remove this function completely and always use the native version?

Having two version of this function around increases the maintenance burden.

Also, using the native version in both cases increases the amount of testing we apply to the native version.

Reply:
Makes sense, in this case it looks like there's no reason not to move to C entirely now. I did not that now.

I wasn't sure which system library to put this in, HTML5 seemed closest..? Or a new one?

Reply:
Good question.  I'd rather not add an entirely new library.. so yes html5 makes sense to me.